src/MSBuildLocator/MSBuildLocator.cs

```diff
@@ -85,6 +96,11 @@ public static void RegisterInstance(VisualStudioInstance instance)
         /// </param>
         public static void RegisterMSBuildPath(string msbuildPath)
         {
+            if (string.IsNullOrWhiteSpace(msbuildPath))
```

Comment:
Maybe check for existence too?

Reply:
I see that there's a list of msbuild dll names. Checking that those dlls actually exist in `msbuildPath` might help debugging when they don't exist.

Reply:
I'm adding the existence check. For checking each dll though, I'm worried there are some cases where potentially not all dlls are required by the caller, so if they're missing then it's fine. The current assembly resolver just ignores missing dlls.